Multisig/Logic/Models/Transaction.swift

```diff
@@ -27,7 +27,7 @@ struct Transaction: Codable {
     // can be modified for WalletConnect transactions
     var nonce: UInt256String
     // computed based on other properties
-    var safeTxHash: HashString?
```

Comment:
guaranteed to be not nil?

Reply:
yes. We calculate it everywhere where we initiate transaction